app/src/js/multimocks.js

```diff
@@ -48,7 +48,8 @@ angular
     'multimocksData',
     function ($q, $http, $httpBackend, multimocksData) {
       var setupHttpBackendForMockResource = function (deferred, mock) {
-        var mockHeaders = multimocksData.getHeaders();
+        var mockHeaders = multimocksData.getHeaders(),
+          uriRegExp = new RegExp('^' + mock.uri + '$');
 
```

Comment:
Did we say we'd we were going to let the ^ and $ be part of the configuration rather than forcing users to make their regexs match the entire url?